databases/week2/more-queries.sql

```diff
@@ -0,0 +1,24 @@
+USE hyf;
+select * from task;
+
+-- 1 
+SELECT  user.name, user.email FROM user
+WHERE user.email REGEXP '@spotify.com$';
+
+-- 2
```

Comment:
This query is supposed to be done on the database from lesson 2, where the tables task and user are connected through a third table called user_task (or something similar). Let me know if there is something that is not clear :)